test/e2e/antreapolicy_test.go

```diff
@@ -211,7 +211,7 @@ func testMutateANPNoRuleName(t *testing.T) {
 		SetAppliedToGroup(map[string]string{"pod": "a"}, nil).
 		SetPriority(10.0).
 		AddIngress(v1.ProtocolTCP, &p80, nil, nil, map[string]string{"pod": "b"}, map[string]string{"ns": "x"},
-			nil, nil, secv1alpha1.RuleActionAllow, "")
+			nil, nil, []ANPRuleAppliedToSpec{}, secv1alpha1.RuleActionAllow, "")
```

Comment:
you changed some of the `[]ANPRuleAppliedToSpec{}` to `nil` as I suggested earlier but not all of them. Intentional?

Reply:
I replaced all []ACNPRuleAppliedToSpec{} but forgot to replace []ANPRuleAppliedToSpec{}... Updated